sonic_y_cable/y_cable.py

```diff
@@ -1350,8 +1356,45 @@ def upgrade_firmware(physical_port, fwfile):
              a string, a path to the binary file which contains the firmware image
```

Comment:
I think we should add a note that the firmware file can be in the format of the vendor's choosing (binary, archive, etc.). But note that it should be one file which contains firmware for all components of the Y-cable.